README.adoc

```diff
@@ -151,7 +151,7 @@ cd ${GOPATH}/src/github.com/kiali/kiali
 make container-build
 ----
 
-This builds the Kiali operator image, too.
+On x86_64 machine, this will build both the Kiali and the Kiali operator image. On aarch64 machine, this will only build the Kiali image.
```

Comment:
aarch64 :question:

Reply:
Created an issue in https://github.com/operator-framework/operator-sdk/issues/2686 in order to enable the building of operator image on arm. Please feel free to leave any comments there.

I checked the method of building from source, in order to achive this, we need to use a few `sed` command to apply changes to their `Makefile` , I personally would not prefer that way, but if the community has request, I will do that change also.

Reply:
> aarch64

See: https://en.wikichip.org/wiki/arm/aarch64